docs/CONTRIBUTING.md

```diff
@@ -20,10 +28,12 @@ make pre-commit
 Required dependencies:
 
 ```
-go get k8s.io/kubernetes
-go get k8s.io/apimachinery
-go get github.com/gogo/protobuf/protoc-gen-gogo
-go get golang.org/x/tools/cmd/goimports
+GO111MODULE=off go get k8s.io/apimachinery           
```

Comment:
Minor - go 16 allows you to “go install” deps, but I’m assuming you have another goal.

Reply:
Yeah, I wanted to download those dependencies to that `go-to-protobuf` would work. This made it work for me.